tutorials/ring_background.ipynb

```diff
@@ -0,0 +1,415 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create exclusion mask\n",
+    "First, we need to create an exclusion mask on the known sources. In this case, we need to mask only `MSH 15-52` but this depends on the sources present in our field of view."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get the geom that we use\n",
+    "geom = analysis.datasets[0].counts.geom\n",
+    "energy_axis = analysis.datasets[0].counts.geom.get_axis_by_name(\"energy\")\n",
+    "geom_image = geom.to_image().to_cube([energy_axis.squash()])\n",
+    "\n",
+    "# Make the exclusion mask\n",
+    "regions = CircleSkyRegion(center=source_pos, radius=0.3 * u.deg)\n",
+    "exclusion_mask = Map.from_geom(geom_image)\n",
+    "exclusion_mask.data = geom_image.region_mask([regions], inside=False)\n",
+    "exclusion_mask.sum_over_axes().plot();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the present analysis, we use a ring with an inner radius of 0.5 deg and width of 0.3 deg."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ring_maker = RingBackgroundMaker(\n",
+    "    r_in=\"0.5 deg\", width=\"0.3 deg\", exclusion_mask=exclusion_mask\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a stacked dataset\n",
+    "Now, we extract the background for each dataset and then stack the maps together to create a sinngle stacked map for further analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%time\n",
+    "stacked_on_off = MapDatasetOnOff.create(geom=geom_image)\n",
+    "for dataset in analysis.datasets:\n",
+    "    dataset_image = (\n",
+    "        dataset.to_image()\n",
+    "    )  # Ring extracting makes sense only for 2D analysis\n",
+    "    dataset_on_off = ring_maker.run(dataset_image)\n",
+    "    stacked_on_off.stack(dataset_on_off)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This `stacked_on_off` has `on` and `off` counts and acceptance maps which we will use in all further analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(stacked_on_off)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compute Li&Ma maps\n",
+    "We need to convolve our maps with an apprpriate smoothing kernel. Since astropy convolution kernels only accept integers, we first convert our required size in degrees to int depending on our pixel size."
```

Comment:
Those are not Li&Ma maps but correlated excess and significance map.

The former is the integrated excess in circles of a given size, while the latter gives the significance of this excess. This significance is computed according to the Li & Ma expression for ON and OFF Poisson measurements and refer to the stat doc.